### PR TITLE
add new lua runtime

### PR DESCRIPTION
**Pull request by ghost**
It combines quick player and old lua runtime.

Main Featrues:
- preview scene editing by Cocos Studio
  -.run Lua project 
- debugging Lua project with Cocos Code IDE
- chang view size on the fly (Desktop)
- writing debug message to file (Desktop)

It bases on this ( https://github.com/cocos2d/cocos2d-x/pull/9618 ) pr.

### CODE REVIEW

File: templates/lua-template-runtime/frameworks/runtime-src/proj.ios_mac/mac/service/PlayerFileDialogServiceMac.h [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+    //
+    // extensions 示例：
+    // extensions = "Lua Script File|*.lua;JSON File|*.json";
+    //
+    // 每一种类型分为“描述”和“扩展名”两部分，用“|”分隔
+    // 扩展名如果有多个，则用“,”分隔
+    // 不同类型用“;”分隔
+    //
+    // 如果 extensions 参数为空，则表示可以选择任何扩展名的文件。
+    //
```
Comment (minggo) [inline templates/lua-template-runtime/frameworks/runtime-src/proj.ios_mac/mac/service/PlayerFileDialogServiceMac.h:31]: Could you please don't use Chinese.

File: templates/lua-template-runtime/frameworks/runtime-src/proj.win32/game.rc [hunk 2]
```diff
@@ -47,30 +47,9 @@ END
 
 IDR_MENU_COCOS MENU
 BEGIN
-    POPUP "&File"
+    POPUP "����(&H)"
```
Comment (minggo) [inline templates/lua-template-runtime/frameworks/runtime-src/proj.win32/game.rc:50]: error code

File: templates/lua-template-runtime/frameworks/runtime-src/proj.win32/game.rc [hunk 3]
```diff
@@ -47,30 +47,9 @@ END
...
-    END
-    POPUP "&Control"
-    BEGIN
-        MENUITEM "Restart(F5)",                 ID_CONTROL_RELOAD
-        MENUITEM "Keep Window Top",             ID_CONTROL_TOP
-    END
-    POPUP "&Help"
-    BEGIN
-        MENUITEM "&About ...",                  ID_HELP_ABOUT
+        MENUITEM "����(&A)",                      ID_HELP_ABOUT
```
Comment (minggo) [inline templates/lua-template-runtime/frameworks/runtime-src/proj.win32/game.rc:52]: error code

File: templates/lua-template-runtime/frameworks/runtime-src/proj.win32/service/stdafx.h [hunk 4]
```diff
@@ -0,0 +1,21 @@
+// stdafx.h : include file for standard system include files,
+// or project specific include files that are used frequently, but
+// are changed infrequently
+//
+
+#pragma once
```
Comment (minggo) [inline templates/lua-template-runtime/frameworks/runtime-src/proj.win32/service/stdafx.h:6]: It is not compiler compatible.

File: templates/lua-template-runtime/frameworks/runtime-src/proj.ios_mac/mac/service/PlayerFileDialogServiceMac.h (hunk 1 above)
Comment (ghost) [inline templates/lua-template-runtime/frameworks/runtime-src/proj.ios_mac/mac/service/PlayerFileDialogServiceMac.h:31]: OK

File: templates/lua-template-runtime/frameworks/runtime-src/proj.win32/service/stdafx.h (hunk 4 above)
Comment (minggo) [inline templates/lua-template-runtime/frameworks/runtime-src/proj.win32/service/stdafx.h:6]: Ok, it is only included on windows.
No problem.

**Pull request merged**